rocprim/include/rocprim/device/device_binary_search.hpp

```diff
@@ -124,6 +161,43 @@ hipError_t lower_bound(void * temporary_storage,
                                          debug_synchronous);
 }
 
+/// \brief Performs a device-level upper bound check.
+///
+/// \par Overview
+/// Runs multiple upper bound checks in parallel (one for each \p needle in \p needles ).
```

Comment:
extra whitespace after needles

Reply:
(As above) I've switched this to using the equivalent: "<tt>needles</tt>", since removing the space causes the closing parenthesis to be considered part of the expression the \p formats.